packages/protocol/test-sol/unit/common/EpochManager.t.sol

```diff
@@ -698,6 +698,19 @@ contract EpochManagerTest_setToProcessGroups is EpochManagerTest {
     assertEq(EpochManager(address(epochManager)).toProcessGroups(), groups.length);
   }
 
+  function test_blocksChilds() public {
```

Comment:
Can we also add the opposite test ? setToProcessGroups and then call  startNextEpochProcess ?

Reply:
isn't this the test that is just bellow in this PR?
`  function test_Reverts_startEpochAgain() public {`

Reply:
not really since it calls

startNextEpochProcess
setToProcessGroups
startNextEpochProcess


What I meant was test with only

setToProcessGroups
startNextEpochProcess

Reply:
Are you supposed to be able to call `setToProcessGroups` before calling `startNextEpochProcess`?

Reply:
no

Reply:
no - it should revert